### PR TITLE
AzureMonitor: docs for multiple subscriptions

### PR DESCRIPTION
**Pull request by daniellee**
- Updates template variable query section to show that can now be used with multiple subscriptions.
- Updated screenshots and text to account for Grafana UI changes
- Also, links to images in Grafana.org instead of the images in the plugin GitHub repo.

On staging: http://staging-docs.grafana.org/features/datasources/azuremonitor/

### CODE REVIEW

File: docs/sources/features/datasources/azuremonitor.md [hunk 1]
```diff
@@ -31,34 +31,37 @@ The datasource can access metrics from four different services. You can configur
...
-8. Paste these two items into the appropriate fields in the Application Insights API Details section:<br/>
-![Application Insights API Details](https://raw.githubusercontent.com/grafana/azure-monitor-datasource/master/src/img/config_3_app_insights_api_details.png)
+5. If you are also using the Azure Log Analytics service, then you need to specify these two config values (or you can reuse the Client Id and Secret from the previous step).
 
-9. Test that the configuration details are correct by clicking on the "Save & Test" button:<br/>
-![Azure Monitor API Details](https://raw.githubusercontent.com/grafana/azure-monitor-datasource/master/src/img/config_4_save_and_test.png)
+   - Client Id (Azure Active Directory -> App Registrations -> Choose your app -> Application ID)
+   - Client Secret ( Azure Active Directory -> App Registrations -> Choose your app -> Keys -> Create a key -> Use client secret)
+
+6. If you are are using Application Insights, then you need two pieces of information from the Azure Portal (see link above for detailed instructions):
```
Comment (marefr) [inline docs/sources/features/datasources/azuremonitor.md]:
```suggestion
6. If you are using Application Insights, then you need two pieces of information from the Azure Portal (see link above for detailed instructions):
```

File: docs/sources/features/datasources/azuremonitor.md [hunk 2]
```diff
@@ -31,34 +31,37 @@ The datasource can access metrics from four different services. You can configur
...
+   - The Subscription Id can be changed per query. Save the datasource and refresh the page to see the list of subscriptions available for the specified Client Id.
 
-8. Paste these two items into the appropriate fields in the Application Insights API Details section:<br/>
-![Application Insights API Details](https://raw.githubusercontent.com/grafana/azure-monitor-datasource/master/src/img/config_3_app_insights_api_details.png)
+5. If you are also using the Azure Log Analytics service, then you need to specify these two config values (or you can reuse the Client Id and Secret from the previous step).
 
-9. Test that the configuration details are correct by clicking on the "Save & Test" button:<br/>
-![Azure Monitor API Details](https://raw.githubusercontent.com/grafana/azure-monitor-datasource/master/src/img/config_4_save_and_test.png)
+   - Client Id (Azure Active Directory -> App Registrations -> Choose your app -> Application ID)
+   - Client Secret ( Azure Active Directory -> App Registrations -> Choose your app -> Keys -> Create a key -> Use client secret)
```
Comment (marefr) [inline docs/sources/features/datasources/azuremonitor.md]:
```suggestion
   - Client Secret (Azure Active Directory -> App Registrations -> Choose your app -> Keys -> Create a key -> Use client secret)
```

File: docs/sources/features/datasources/azuremonitor.md [hunk 3]
```diff
@@ -222,25 +238,27 @@ AzureActivity
...
 
 - `$__timeFilter(datetimeColumn)` - Expands to
-    `datetimeColumn  ≥ datetime(2018-06-05T18:09:58.907Z) and`
-    `datetimeColumn ≤ datetime(2018-06-05T20:09:58.907Z)` where the from and to datetimes are from the Grafana time picker.
+  `datetimeColumn ≥ datetime(2018-06-05T18:09:58.907Z) and`
+  `datetimeColumn ≤ datetime(2018-06-05T20:09:58.907Z)` where the from and to datetimes are from the Grafana time picker.
 
-- `$__escapeMulti($myVar)` - is to be used with multi-value template variables that contains illegal characters. If $myVar has the value  `'\\grafana-vm\Network(eth0)\Total','\\hello!'`, it expands to: `@'\\grafana-vm\Network(eth0)\Total', @'\\hello!'`. If using single value variables there no need for this macro, simply escape the variable inline instead - `@'\$myVar'`
+- `$__escapeMulti($myVar)` - is to be used with multi-value template variables that contains illegal characters. If \$myVar has the value `'\\grafana-vm\Network(eth0)\Total','\\hello!'`, it expands to: `@'\\grafana-vm\Network(eth0)\Total', @'\\hello!'`. If using single value variables there no need for this macro, simply escape the variable inline instead - `@'\$myVar'`
```
Comment (marefr) [inline docs/sources/features/datasources/azuremonitor.md]: Should `\$myVar` really render as `\$myVar` in all occurrences? 

Looks a bit weird on staging to me: http://staging-docs.grafana.org/features/datasources/azuremonitor/#azure-log-analytics-macros

**Review by marefr — CHANGES_REQUESTED** [review]
Looks good, but added a few suggestion and comments.

File: docs/sources/features/datasources/azuremonitor.md (hunk 3 above)
Comment (daniellee) [reply docs/sources/features/datasources/azuremonitor.md]: No - nothing to do with this change but can fix.

**Review by marefr — APPROVED** [review]
Great

**Pull request merged**